### PR TITLE
BZ2012651: Push graph data image in restricted network

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Bug: https://bugzilla.redhat.com/show_bug.cgi?id=2012651

Releases: 4.6, 4.7, 4.8, 4.9, 4.10

Preview link: https://deploy-preview-39096--osdocs.netlify.app/openshift-enterprise/latest/updating/installing-update-service.html#update-service-graph-data_update-service

@shellyyang1989 - Kindly help with the QE review.

### CODE REVIEW

**Comment by shellyyang1989** [comment]
As we've made an agreement with devel in slack about the change. So lgtm.

File: modules/update-service-graph-data.adoc [hunk 1]
```diff
@@ -29,3 +29,8 @@ $ podman build -f ./Dockerfile -t registry.example.com/openshift/graph-data:late
 ----
 $ podman push registry.example.com/openshift/graph-data:latest
 ----
++
+[NOTE]
+====
+For pushing a graph data image to a local registry in a restricted network, copy the graph-data container image created in the previous step to a repository that is accessible to the OpenShift Update Service. You can run `oc image mirror --help` for available options.
```
Comment (sagidlow) [inline modules/update-service-graph-data.adoc:35]:
```suggestion
You can push a graph data image to a local registry in a restricted network. Copy the `graph-data` container image created in the previous step to a repository that is accessible to the OpenShift Update Service. Run `oc image mirror --help` for available options.
```

Suggested for simpler for reading and minimalistic writing.

**Pull request merged**